src/services/tecnico.js

```diff
@@ -417,3 +417,28 @@ export const createPDF = async (technician, data) => {
 
   doc.save(`ROMANEIO_${moment(data).format('L')}.pdf`)
 }
+
+export const getTechnicianById = async (id) => {
+  const storeObject = store.getState()
+
+  const headers = {
+    token: storeObject.auth.token,
+    username: storeObject.auth.username,
+  }
+
+  let response = {}
+
+  await axios.get(`${BACKEND_URL}/api/technician/getById`, { headers: headers, params: { id } }).then(
```

Comment:
aqui usamos a intancia que criamos no arquivo do request e usamos aqui só importar e usar
```suggestion
  await axiosInstance.get(`${BACKEND_URL}/api/technician/getById`, { headers: headers, params: { id } })
```